packages/preview-middleware-client/src/adp/quick-actions/fe-v4/create-table-action.ts

```diff
@@ -24,28 +25,27 @@ export class AddTableActionQuickAction extends TableQuickActionDefinitionBase im
 
     async execute(path: string): Promise<FlexCommand[]> {
         const index = this.tableMap[path];
-        const smartTablesToolbarAction = getRelevantControlFromActivePage(
-            this.context.controlIndex,
-            this.context.view,
-            [TOOLBAR_ACTION]
-        );
-        for (let i = 0; i < smartTablesToolbarAction.length; i++) {
-            if (i === index) {
-                const section = getControlById(smartTablesToolbarAction[i].getId());
-                const controlOverlay = OverlayUtil.getClosestOverlayFor(section);
-                if (controlOverlay) {
-                    controlOverlay.setSelected(true);
-                    await DialogFactory.createDialog(
-                        controlOverlay,
-                        this.context.rta,
-                        DialogNames.ADD_FRAGMENT,
-                        undefined,
-                        {
-                            aggregation: 'actions',
-                            title: 'QUICK_ACTION_ADD_CUSTOM_TABLE_ACTION'
-                        }
-                    );
-                }
+        const actionToolbars = getRelevantControlFromActivePage(this.context.controlIndex, this.context.view, [
```

Comment:
Why are we using toolbar and not the tables `actions` aggregation? According to the documentation everything under content aggregation should not be modified as it may change and break when using a different UI5 version. We probably should avoid creating such changes.